plugins/remotefortressreader/remotefortressreader.cpp

```diff
@@ -2793,6 +3078,11 @@ static command_result GetPartialCreatureRaws(color_ostream &stream, const ListRe
 
             CopyMat(send_tissue->mutable_material(), orig_tissue->mat_type, orig_tissue->mat_index);
         }
+
+        for (int i = 0; i <= ENUM_LAST_ITEM(creature_raw_flags); i++)
```

Comment:
Should use FOR_ENUM_ITEMS or at least ENUM_FIRST_ITEM for consistency (FOR_ENUM_ITEMS would avoid the cast below)